homeworks/edgar_meloyan/Homework 1/BitwiseOperators.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
```

Comment:
you solved different problem than required by https://www.hackerrank.com/challenges/bitwise-operators-in-c/problem?h_r=internal-search&isFullScreen=true